src/main/resources/application.properties

```diff
@@ -10,9 +10,10 @@ spring.profiles.active=dev
 spring.datasource.url=${DB_URL}
 spring.datasource.username=${DB_USERNAME}
 spring.datasource.password=${DB_PASSWORD}
-spring.datasource.driver-class-name=${DB_DRIVER_CLASS_NAME}
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
 spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
```

Comment:
what is this used for?

Reply:
It tells hibernate that we are using MYSQL syntax so it does not confuse it with PostgreSQL or Oracle. If not defined, hibernate tries to identify it based on the database used, however it was throwing and error when this was not specified. So I specified it to get rid of the error.